runtime/v2/shim/shim_unix.go

```diff
@@ -37,14 +37,30 @@ import (
 // sub-reaper so that the container processes are reparented
 func setupSignals(config Config) (chan os.Signal, error) {
 	signals := make(chan os.Signal, 32)
-	smp := []os.Signal{unix.SIGTERM, unix.SIGINT, unix.SIGPIPE}
+	smp := []os.Signal{unix.SIGPIPE}
 	if !config.NoReaper {
 		smp = append(smp, unix.SIGCHLD)
 	}
 	signal.Notify(signals, smp...)
 	return signals, nil
 }
 
+func handleExitSignals(ctx context.Context, logger *logrus.Entry, cancel context.CancelFunc) {
+	ch := make(chan os.Signal, 32)
+	signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
We need SIGRTMIN handling as @AkihiroSuda and @thaJeztah mentioned.

Reply:
I'm not sure I understand this... wouldn't systemd itself receives `SIGRTMIN+24`, but shouldn't the shim receive a normal exit signal?

Reply:
@kzys @AkihiroSuda ^^ ptal